docs/faq.md

```diff
@@ -16,3 +16,7 @@ If you see the annotation `kubectl.kubernetes.io/last-applied-configuration` on
 ### I don't use helm, how can I do in cluster checks?
 
 Currently, the only in-cluster check we are confident in supporting is helm. If your deployment method can generate yaml manifests for kubernetes, you should be able to use the `detect` or `detect-files` functionality described below after the manifest files have been generated.
+
+### I updated the API version of an object, but pluto still reports that the apiVersion needs to be updated.
+
+Pluto looks at the API Versions of objects in releases that are in a `Deployed` state, and Helm has an issue where it might list old revisions of a release as still being in a `Deployed` state. To fix this, look at the release revision history with `helm history <release name>`, and determine if older releases still show a `Deployed` state. If so, delete the Helm release secret(s) (or configMap with Helm v2) associated with the revision number(s). For example, `helm delete secret sh.helm.release.v1.my-release.v10` where `10` corresponds to the release number. Then run Pluto again to see if the object has been removed from the report.
```

Comment:
We no longer support Helm2, so I removed it.

```suggestion
Pluto looks at the API Versions of objects in releases that are in a `Deployed` state, and Helm has an issue where it might list old revisions of a release as still being in a `Deployed` state. To fix this, look at the release revision history with `helm history <release name>`, and determine if older releases still show a `Deployed` state. If so, delete the Helm release secret(s) associated with the revision number(s). For example, `helm delete secret sh.helm.release.v1.my-release.v10` where `10` corresponds to the release number. Then run Pluto again to see if the object has been removed from the report.
```

Reply:
There is technically another option to edit the status of that particular release. Not sure if we want to mention that here.